admin/api.go

```diff
@@ -165,6 +163,12 @@ func (a *AdminAPI) GetConnectors() ([]connector.ConnectorConfig, error) {
 }
 
 func mapError(e error) error {
+	switch t := e.(type) {
+	case client.ValidationError:
```

Comment:
Maybe put it on ValidationError? Whichever works best. If it's tested that might be enough.